analyticsdataserver/settings/base.py

```diff
@@ -309,6 +312,12 @@
 # base url to generate link to user api
 LMS_USER_ACCOUNT_BASE_URL = None
 
+# storage settings for report downloads
+DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+MEDIA_ROOT = '/edx/app/analytics_api/analytics_api/static/reports'
```

Comment:
Can we use DJANGO_ROOT here instead? If we are running the server outside of the analytics devstack this will likely fail due to permissions or because the directory doesn't exist.

Something like: `normpath(join(DJANGO_ROOT, 'static', 'reports'))`